crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/utils/helpers.py

```
@@ -0,0 +1,16 @@
+def helper_function():
+    """A simple helper function."""
+    return "helper_result"
+
+
+class UtilityClass:
+    """A simple utility class."""
+
+    def __init__(self, value):
+        self.value = value
+
+    def get_value(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_value`

(ANN201)